pkg/agent/dozer/bcm/plan.go

```diff
@@ -405,6 +409,64 @@ func planFabricConnections(agent *agentapi.Agent, spec *dozer.Spec) error {
 		},
 	}
 
+	// for spines, we want to advertise every /32 in the service IP range
+	// for leaves, we want to advertise only the switch's own VTEP IP
+	var servicePrefix string
```

Comment:
why is it different between spines and leafs? shouldn't we always advertise protocol IP regardless of the role and on top of it VTEP IP for leafs?